src/styles/main.scss

```diff
@@ -1,3 +1,220 @@
+$baseButtonColor: #00acdc;
+$baseTextColor: #060b35;
+$headerLinkColor: #060b35;
+$hoverHeaderLinkColor: #00acdc;
+$extraTextColor: #616070;
+$navLinkHeight: 60px;
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```

Comment:
union fonts with font-family in body, and delete here

Reply:
done